src/layout/layout.rs

```diff
@@ -720,6 +720,16 @@ fn configure_constraints(
 ) -> Result<(), AddConstraintError> {
     for (&constraint, &element) in constraints.iter().zip(segments.iter()) {
         match constraint {
+            Constraint::MinMax(min, max) => {
+                solver.add_constraint(element.has_min_size(min, MIN_SIZE_GE))?;
+                solver.add_constraint(element.has_max_size(max, MAX_SIZE_LE))?;
+                if flex.is_legacy() {
+                    solver.add_constraint(element.has_int_size(min, MIN_SIZE_EQ))?;
```

Comment:
imo we don't need to special case this one because `MinMax` didn't exist before, so it there's no need to implement it the way `Min` behaves in `Flex::Legacy`.

Additionally, these two constraints will have the opposite effect, i.e. try to make the size `min` versus try to make the size `max`. Meaning only one of them can succeed. I'm not sure if that's what we want.

```
solver.add_constraint(element.has_int_size(min, MIN_SIZE_EQ))?;
solver.add_constraint(element.has_int_size(max, MAX_SIZE_EQ))?;
```

With the inclusion of this constraint, I believe it is always going to favor making the size `max`:

```
solver.add_constraint(element.has_size(area, FILL_GROW))?;
```

I _think_ we want `MinMax` to behave this way, so we should probably drop this line:

```
solver.add_constraint(element.has_int_size(min, MIN_SIZE_EQ))?;
```

fwiw, I'm not asking you to make these changes right now, just commenting for posterity. I think we want to use the flex modes and experiment with them to see what gaps they are and see how they can be filled. Because it might be possible we need to add more than just a `MinMax` constraint.